uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -4,210 +4,206 @@
+                  ng-show="$ctrl.uiOptions.multipleDays">
+                <div class="col-xs-12 form-horizontal">
+                  <div class="form-group">
+                    <label class="col-xs-4 control-label">To Date
                   <i class="fa fa-asterisk"></i>
                 </label>
-                  <div class="col-xs-4">
-                    <div class="input-group">
-                      <input type="text"
-                          class="form-control"
-                          uib-datepicker-popup
-                          datepicker-options="$ctrl.uiOptions.datePickerOptions"
-                          show-button-bar="false"
-                          ng-model="$ctrl.uiOptions.toDate"
-                          is-open="$ctrl.uiOptions.showDatePickerTo"
-                          ng-change="$ctrl.onDateChange($ctrl.uiOptions.toDate, 'to')" />
-                      <span class="input-group-btn">
+                    <div class="col-xs-4">
+                      <div class="input-group">
+                        <input type="text"
+                            class="form-control"
+                            uib-datepicker-popup
+                            datepicker-options="$ctrl.uiOptions.datePickerOptions"
+                            show-button-bar="false"
+                            ng-model="$ctrl.uiOptions.toDate"
+                            is-open="$ctrl.uiOptions.showDatePickerTo"
+                            ng-change="$ctrl.onDateChange($ctrl.uiOptions.toDate, 'to')" />
+                        <span class="input-group-btn">
                         <button type="button" class="btn btn-default fa fa-calendar"
                                 ng-click="$ctrl.uiOptions.showDatePickerTo = true"></button>
                       </span>
+                      </div>
                     </div>
-                  </div>
 
-                  <div class="col-xs-4">
-                    <crm-loading show="!$ctrl.loading.toDayTypes">
-                      <div class="chr_custom-select chr_custom-select--full">
-                        <select name="toDayTypeSelect"
-                            ng-disabled="$ctrl.leaveRequestToDayTypes.length == 1"
-                            ng-options="option.label for option in $ctrl.leaveRequestToDayTypes track by option.name"
-                            ng-model="$ctrl.uiOptions.selectedToType"
-                            ng-change="$ctrl.calculateBalanceChange()">
-                      <option value="">Select</option>
-                    </select>
-                      </div>
-                    </crm-loading>
+                    <div class="col-xs-4">
+                      <crm-loading show="!$ctrl.loading.toDayTypes">
+                        <div class="chr_custom-select chr_custom-select--full">
+                          <select name="toDayTypeSelect"
+                              ng-disabled="$ctrl.leaveRequestToDayTypes.length == 1"
```

Comment:
`$ctrl.leaveRequestToDayTypes.length === 1`
=== should be used for Integer comparison